src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -593,7 +593,7 @@ public boolean isLoggable(LogRecord record) {
 
         /** List of file to validate. */
         @Parameters(arity = "1..*", description = "One or more source files to verify")
-        private final List<File> files = new ArrayList<>();
+        private List<File> files;
```

Comment:
This change is because picocli never used the original files list.
`files` is required, so you can't get by with using the empty list. When atleast 1 file is given, picocli overwrites `files` with a new list using reflection, even though the field was `final`. It never uses the default value, and thats why it can be removed.

Reply:
Confirmed